### PR TITLE
reference: add 11 new documents for TiFlash

### PR DESCRIPTION
**Pull request by ilovesoup**
### What is changed, added or deleted? (Required)

Add TiFlash contents

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v4.0 changes for now)
- [x] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

**If you select two or more versions from above**, to trigger the bot to cherry-pick this PR to your desired release version branch(es), you **must** add corresponding labels such as **needs-cherry-pick-3.1**, **needs-cherry-pick-3.0**, and **needs-cherry-pick-2.1**.

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: reference/tiflash/maintain.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    echo 'config show replication' | /path/to/pd-ctl -u http://<pd-ip>:<pd-port>
+    ```
+
+    预期结果为 `"enable-placement-rules": "true"`。
+
+2. 检查 TiFlash 进程是否正常。
```
Comment (yikeke) [inline reference/tiflash/maintain.md:80]: 有没有检查的命令可以提供？

File: reference/tiflash/upgrade.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+category: reference
+---
+
+# 升级 TiFlash 节点
+
+升级 TiFlash 节点的步骤如下：
+
+1. 下载新版 TiFlash binary 到中控机，方式有如下两种：
+
+    - 可以更新到新版 TiDB Ansible，然后执行 `ansible-playbook local_prepare.yml` 命令。
```
Comment (yikeke) [inline reference/tiflash/upgrade.md:12]:
```suggestion
    - 可以更新到新版 TiDB Ansible，然后执行 `ansible-playbook local_prepare.yml` 命令。
```
更新到新版 TiDB Ansible 有参考的文档或可以直接提供的命令吗

File: reference/tiflash/upgrade.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    - 或者手动下载 TiFlash binary 并覆盖到 `resource/bin/tiflash`。
```
Comment (yikeke) [inline reference/tiflash/upgrade.md:13]:
```suggestion
    - 或者手动下载 TiFlash binary 并覆盖到 `resource/bin/tiflash`。
```
TiFlash binary 的下载地址能提供下吗

**Comment by yikeke** [comment]
Preview the 11 .md files here:

docs-cn/overview.md 
https://github.com/pingcap/docs-cn/blob/050e322b2d7cecab28d0d3c8e03d12564ab5327a/reference/tiflash/overview.md

docs-cn/alert-rules.md
https://github.com/pingcap/docs-cn/blob/5051a175a2dcf31ce2fbc75ba1ca3f907d5e99bd/reference/tiflash/alert-rules.md

docs-cn/configuration.md 
https://github.com/pingcap/docs-cn/blob/5051a175a2dcf31ce2fbc75ba1ca3f907d5e99bd/reference/tiflash/configuration.md

docs-cn/deploy.md 
https://github.com/pingcap/docs-cn/blob/050e322b2d7cecab28d0d3c8e03d12564ab5327a/reference/tiflash/deploy.md#在原有-tidb-集群上新增-tiflash-组件

docs-cn/faq.md 
https://github.com/pingcap/docs-cn/blob/050e322b2d7cecab28d0d3c8e03d12564ab5327a/reference/tiflash/faq.md

docs-cn/maintain.md 
https://github.com/pingcap/docs-cn/blob/050e322b2d7cecab28d0d3c8e03d12564ab5327a/reference/tiflash/maintain.md

docs-cn/monitor.md 
https://github.com/pingcap/docs-cn/blob/050e322b2d7cecab28d0d3c8e03d12564ab5327a/reference/tiflash/monitor.md

docs-cn/scale.md 
https://github.com/pingcap/docs-cn/blob/050e322b2d7cecab28d0d3c8e03d12564ab5327a/reference/tiflash/scale.md

docs-cn/tune-performance.md 
https://github.com/pingcap/docs-cn/blob/050e322b2d7cecab28d0d3c8e03d12564ab5327a/reference/tiflash/tune-performance.md

docs-cn/upgrade.md
https://github.com/pingcap/docs-cn/blob/4939b1560445c6ad1bd8555bba555e1653f27cc4/reference/tiflash/upgrade.md

docs-cn/use-tiflash.md 
https://github.com/pingcap/docs-cn/blob/c82960f44179dfca103c12532f6a8e3a7d9acbdd/reference/tiflash/use-tiflash.md

File: reference/tiflash/configuration.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+可通过 [pd-ctl](/reference/tools/pd-control.md) (tidb-ansible 目录下的 `resources/bin` 包含对应的二进制文件) 调整参数：
+
+- [`replica-schedule-limit`](/reference/configuration/pd-server/configuration-file.md#replica-schedule-limit)：用来控制 replica 相关 operator 的产生速度（涉及到下线、补副本的操作都与该参数有关）
+
+    > **注意：**
+    >
+    > 不要超过 `region-schedule-limit`，否则会影响正常 TiKV 之间的 Region 调度。
+
+- `store-balance-rate`：用于限制每个 store 的调度速度
```
[the remark this answers is not in the document]
Comment (ilovesoup) [reply reference/tiflash/configuration.md:20]: 明天我问下pd吧，我搜了下看起来pd也没说明，这个建议是pd给个link然后我们链过去，这不是tiflash部署本身的configuration，有全局影响。

File: reference/tiflash/upgrade.md (hunk 2 above)
Comment (ilovesoup) [reply reference/tiflash/upgrade.md:12]: 是 git 命令，我让同事加一下吧。

File: reference/tiflash/upgrade.md (hunk 3 above)
Comment (ilovesoup) [reply reference/tiflash/upgrade.md:13]: 这是按需提供的，并不会统一给，比如临时有修复需要升级。

File: reference/tiflash/configuration.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (yikeke) [reply reference/tiflash/configuration.md:20]: 好的

File: reference/tiflash/upgrade.md (hunk 3 above)
Comment (yikeke) [reply reference/tiflash/upgrade.md:13]: ok，那这里不用改了

File: reference/tiflash/maintain.md [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+
+1. 在 TiDB 客户端中针对所有副本数大于集群剩余 TiFlash 节点数的表执行：
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    alter table <db-name>.<table-name> set tiflash replica 0;
+    ```
+
+2. 等待相关表的 TiFlash 副本被删除（按照[“查看表同步进度”](/reference/tiflash/use-tiflash.md#查看表同步进度)一节操作，查不到相关表的同步信息时即为副本被删除）。
```
Comment (ran-huang) [inline reference/tiflash/maintain.md:50]:
```suggestion
2. 等待相关表的 TiFlash 副本被删除（按照[查看表同步进度](/reference/tiflash/use-tiflash.md#查看表同步进度)一节操作，查不到相关表的同步信息时即为副本被删除）。
```

File: reference/tiflash/maintain.md [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+
+## TiFlash 故障处理
+
+本节介绍了一些 TiFlash 常见问题、原因及解决办法。
+
+### TiFlash 副本始终处于不可用状态
+
+该问题一般由于配置错误或者环境问题导致 TiFlash 处于异常状态，可以先通过以下步骤定位问题组件：
+
+1. 检查 PD 是否开启 Placement Rules 功能（开启方法见[“在原有 TiDB 集群上新增 TiFlash 组件”](/reference/tiflash/deploy.md#在原有-tidb-集群上新增-tiflash-组件)的第 2 步）：
```
Comment (ran-huang) [inline reference/tiflash/maintain.md:70]:
```suggestion
1. 检查 PD 是否开启 Placement Rules 功能（开启方法见[在原有 TiDB 集群上新增 TiFlash 组件](/reference/tiflash/deploy.md#在原有-tidb-集群上新增-tiflash-组件)的第 2 步）：
```

File: reference/tiflash/maintain.md [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+
+如果在 TiFlash 上的负载压力过大，会导致 TiFlash 数据同步落后，部分查询可能会返回 `Region Unavailable` 的错误。
+
+在这种情况下，可以通过增加 TiFlash 节点数分担负载压力。
+
+### 数据文件损坏
+
+可依照如下步骤进行处理：
+
+1. 参照[“下线 TiFlash 节点”](/reference/tiflash/maintain.md#下线-tiflash-节点)一节下线对应的 TiFlash 节点。
```
Comment (ran-huang) [inline reference/tiflash/maintain.md:122]:
```suggestion
1. 参照[下线 TiFlash 节点](/reference/tiflash/maintain.md#下线-tiflash-节点)一节下线对应的 TiFlash 节点。
```

File: reference/tiflash/maintain.md [hunk 8]
```diff
@@ -0,0 +1,147 @@
...
+|---------------|---------------------|
+| [ 23 ] <Information> KVStore: Start to persist [region 47, applied: term 6 index 10] | 在 TiFlash 中看到类似日志代表数据开始同步（该日志开头方括号内的数字代表线程号，下同） |
+| [ 30 ] <Debug> CoprocessorHandler: grpc::Status DB::CoprocessorHandler::execute() | Handling DAG request，该日志代表 TiFlash 开始处理一个 Coprocessor 请求 |
+| [ 30 ] <Debug> CoprocessorHandler: grpc::Status DB::CoprocessorHandler::execute() | Handle DAG request done，该日志代表 TiFlash 完成 Coprocessor 请求的处理 |
+
+你可以找到一个 Coprocessor 请求的开始或结束，然后通过日志前面打印的线程号找到该 Coprocessor 请求的其他相关日志。
+
+## TiFlash 系统表
+
+information_schema.tiflash_replica 系统表的列名及含义如下：
```
Comment (ran-huang) [inline reference/tiflash/maintain.md:138]:
```suggestion
`information_schema.tiflash_replica` 系统表的列名及含义如下：
```

File: reference/tiflash/overview.md [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+
+TiFlash 提供列式存储，且拥有借助 ClickHouse 高效实现的协处理器层。除此以外，它与 TiKV 非常类似，依赖同样的 Multi-Raft 体系，以 Region 为单位进行数据复制和分散（详情见[《说存储》](https://pingcap.com/blog-cn/tidb-internal-1/)一文）。
+
+TiFlash 以低消耗不阻塞 TiKV 写入的方式，实时复制 TiKV 集群中的数据，并同时提供与 TiKV 一样的一致性读取，且可以保证读取到最新的数据。TiFlash 中的 Region 副本与 TiKV 中完全对应，且会跟随 TiKV 中的 Leader 副本同时进行分裂与合并。
+
+TiFlash 可以兼容 TiDB 与 TiSpark，用户可以选择使用不同的计算引擎。
+
+TiFlash 推荐使用和 TiKV 不同的节点以做到 Workload 隔离，但在无业务隔离的前提下，也可以选择与 TiKV 同节点部署。
+
+TiFlash 暂时无法直接接受数据写入，任何数据必须先写入 TiKV 再同步到 TiFlash。TiFlash 以 learner 角色接入 TiDB 集群，TiFlash 支持表粒度的数据同步，部署后默认情况下不会同步任何数据，需要按照[“按表构建 TiFlash 副本”](/reference/tiflash/use-tiflash.md#按表构建-tiflash-副本)一节完成指定表的数据同步。
```
Comment (ran-huang) [inline reference/tiflash/overview.md:24]:
```suggestion
TiFlash 暂时无法直接接受数据写入，任何数据必须先写入 TiKV 再同步到 TiFlash。TiFlash 以 learner 角色接入 TiDB 集群，TiFlash 支持表粒度的数据同步，部署后默认情况下不会同步任何数据，需要按照[按表构建 TiFlash 副本](/reference/tiflash/use-tiflash.md#按表构建-tiflash-副本)一节完成指定表的数据同步。
```

File: reference/tiflash/overview.md [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+
+TiFlash 主要包含三个组件，除了主要的存储引擎组件，另外包含 tiflash proxy 和 pd buddy 组件，其中 tiflash proxy 主要用于处理 multi raft 协议通信的相关工作，pd buddy 负责与 PD 协同工作，将 TiKV 数据按表同步到 TiFlash。
```
Comment (ran-huang) [inline reference/tiflash/overview.md:26]:
```suggestion
TiFlash 主要包含三个组件，除了主要的存储引擎组件，另外包含 tiflash proxy 和 pd buddy 组件，其中 tiflash proxy 主要用于处理 Multi-Raft 协议通信的相关工作，pd buddy 负责与 PD 协同工作，将 TiKV 数据按表同步到 TiFlash。
```

File: reference/tiflash/scale.md [hunk 11]
```diff
@@ -0,0 +1,126 @@
...
+
+# TiFlash 集群扩缩容
+
+本文介绍扩缩容 TiFlash 集群节点的步骤。
+
+## 扩容 TiFlash 节点
+
+以在节点 192.168.1.1 上部署 TiFlash 为例，扩容该 TiFlash 节点的步骤如下。
+
+1. 编辑 `inventory.ini` 文件，添加该 TiFlash 节点信息（目前只支持 ip，不支持域名）
```
Comment (ran-huang) [inline reference/tiflash/scale.md:14]:
```suggestion
1. 编辑 `inventory.ini` 文件，添加该 TiFlash 节点信息（目前只支持 ip，不支持域名）：
```

File: reference/tiflash/scale.md [hunk 12]
```diff
@@ -0,0 +1,126 @@
...
+
+## 缩容 TiFlash 节点
+
+以停止 192.168.1.1 节点的服务为例，缩容该 TiFlash 节点的步骤如下。
+
+> **注意：**
+>
+> 本节介绍的下线流程不会删除下线节点上的数据文件，如需再次上线，请先手动删除。
+
+1. 首先参考[“下线 TiFlash 节点”](/reference/tiflash/maintain.md#下线-tiflash-节点)章节，对要进行缩容的 TiFlash 节点进行下线操作。
```
Comment (ran-huang) [inline reference/tiflash/scale.md:96]:
```suggestion
1. 首先参考[下线 TiFlash 节点](/reference/tiflash/maintain.md#下线-tiflash-节点)章节，对要进行缩容的 TiFlash 节点进行下线操作。
```

File: reference/tiflash/scale.md [hunk 13]
```diff
@@ -0,0 +1,126 @@
...
+
+2. 使用 Grafana 或者 pd-ctl 检查节点是否下线成功（下线需要一定时间）。
+
+3. 等待 tiflash 对应的 `store` 消失，或者 `state_name` 变成 `Tomestone` 后，执行如下命令关闭 tiflash 进程：
```
Comment (ran-huang) [inline reference/tiflash/scale.md:100]:
```suggestion
3. 等待 TiFlash 对应的 `store` 消失，或者 `state_name` 变成 `Tomestone` 后，执行如下命令关闭 TiFlash 进程：
```

File: reference/configuration/pd-server/configuration-file.md [hunk 14]
```diff
@@ -236,6 +236,11 @@ PD 配置文件比命令行参数支持更多的选项。你可以在 [conf/conf
 + 关闭隔离级别检查的开关，当设置为 true 时，PD 不会通过调度来提升 Region 副本的隔离级别。
 + 默认：false
 
+### `store-balance-rate`
+
++ 控制 tikv 每分钟最多只允许做 add peer 相关操作的次数。
```
Comment (yikeke) [inline reference/configuration/pd-server/configuration-file.md:241]:
```suggestion
+ 控制 TiKV 每分钟最多允许做 add peer 相关操作的次数。
```

File: reference/tiflash/use-tiflash.md [hunk 15]
```diff
@@ -0,0 +1,168 @@
...
+
+对于创建了 TiFlash 副本的表，TiDB 优化器会自动根据代价估算选择是否使用 TiFlash 副本。具体有没有选择 TiFlash 副本，可以通过 `explain analyze` 语句查看，见下图：
+
+![tidb-display](/media/tiflash/tidb-display.png)
+
+`cop[tiflash]` 表示该任务会发送至 TiFlash 进行处理。如果没有选择 TiFlash 副本，可尝试通过 `analyze table` 语句更新统计信息后，再查看 `explain analyze` 结果。
+
+### Engine 隔离
+
+Engine 隔离是通过配置变量来指定所有的查询均使用指定 engine 的副本，可选 engine 为 tikv 和 tiflash，分别有 2 个配置级别：
```
Comment (ran-huang) [inline reference/tiflash/use-tiflash.md:93]:
```suggestion
Engine 隔离是通过配置变量来指定所有的查询均使用指定 engine 的副本，可选 engine 为 TiKV 和 TiFlash，分别有 2 个配置级别：
```

File: reference/tiflash/use-tiflash.md [hunk 16]
```diff
@@ -0,0 +1,168 @@
...
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    set SESSION tidb_isolation_read_engines = "逗号分隔的 engine list";
+    ```
+
+    会话级别的默认配置继承自 TiDB 实例级别的配置，见 2。
+
+2. TiDB 实例级别，即 INSTANCE 级别，和会话级别配置是交集关系。比如实例级别配置了 "tikv, tiflash"，而会话级别配置了 "tikv"，则只会读取 tikv。
```
Comment (ran-huang) [inline reference/tiflash/use-tiflash.md:113]:
```suggestion
2. TiDB 实例级别，即 INSTANCE 级别，和会话级别配置是交集关系。比如实例级别配置了 "tikv, tiflash"，而会话级别配置了 "tikv"，则只会读取 TiKV。
```

File: reference/tiflash/use-tiflash.md [hunk 17]
```diff
@@ -0,0 +1,168 @@
...
+    在 TiDB 的配置文件添加如下配置项：
+
+    ```
+    [isolation-read]
+    engines = ["tikv", "tiflash"]
+    ```
+
+    实例级别的默认配置为 `["tikv", "tiflash"]`。
+
+当 engine 配置为 "tikv, tiflash"，即可以同时读取 tikv 和 tiflash 副本，优化器会自动选择。指定了 engine 后，对于查询中的表没有对应 engine 副本的情况（因为 tikv 副本是必定存在的，因此只有配置了 engine 为 tiflash 而 tiflash 副本不存在这一种情况），查询会报该表不存在该 engine 副本的错。
```
Comment (ran-huang) [inline reference/tiflash/use-tiflash.md:124]:
```suggestion
当 engine 配置为 "tikv, tiflash"，即可以同时读取 TiKV 和 TiFlash 副本，优化器会自动选择。指定了 engine 后，对于查询中的表没有对应 engine 副本的情况（因为 TiKV 副本是必定存在的，因此只有配置了 engine 为 TiFlash 而 TiFlash 副本不存在这一种情况），查询会报该表不存在该 engine 副本的错。
```

File: reference/tiflash/use-tiflash.md [hunk 18]
```diff
@@ -0,0 +1,168 @@
...
+
+### 手工 Hint
+
+手工 hint 可以强制 TiDB 对于某张或某几张表使用 TiFlash 副本，其优先级低于 engine 隔离，如果 hint 中指定的引擎不在 engine 列表中，会返回 warning，使用方法为：
```
Comment (ran-huang) [inline reference/tiflash/use-tiflash.md:128]:
```suggestion
手工 Hint 可以强制 TiDB 对于某张或某几张表使用 TiFlash 副本，其优先级低于 engine 隔离，如果 Hint 中指定的引擎不在 engine 列表中，会返回 warning，使用方法为：
```

File: reference/tiflash/use-tiflash.md [hunk 19]
```diff
@@ -0,0 +1,168 @@
...
+
+```sql
+select /*+ read_from_storage(tiflash[table_name]) */ ... from table_name;
+```
+
+Engine 隔离的优先级高于 CBO 与 Hint，Hint 优先级高于代价估算，即代价估算仅会选取指定 engine 的副本。
+
+## 使用 TiSpark 读取 TiFlash
+
+TiSpark 目前提供类似 TiDB 中 engine 隔离的方式读取 TiFlash，方式是通过配置参数 `spark.tispark.use.tiflash` 为 true（或 false）。
```
Comment (ran-huang) [inline reference/tiflash/use-tiflash.md:140]:
```suggestion
TiSpark 目前提供类似 TiDB 中 engine 隔离的方式读取 TiFlash，方式是通过配置参数 `spark.tispark.use.tiflash` 为 `true`（或 `false`）。
```

File: reference/tiflash/use-tiflash.md [hunk 20]
```diff
@@ -0,0 +1,168 @@
...
+
+> **注意：**
+>
+> 设为 true 时，所有查询的表都会只读取 TiFlash 副本，设为 `false` 则只读取 TiKV 副本。设为 `true` 时，要求查询所用到的表都必须已创建了 TiFlash 副本，对于未创建 TiFlash 副本的表的查询会报错。
```
Comment (ran-huang) [inline reference/tiflash/use-tiflash.md:144]:
```suggestion
> 设为 `true` 时，所有查询的表都会只读取 TiFlash 副本，设为 `false` 则只读取 TiKV 副本。设为 `true` 时，要求查询所用到的表都必须已创建了 TiFlash 副本，对于未创建 TiFlash 副本的表的查询会报错。
```

File: reference/tiflash/use-tiflash.md [hunk 21]
```diff
@@ -0,0 +1,168 @@
...
+
+可以使用以下任意一种方式进行设置：
+
+1. 在 `spark-defaults.conf` 文件中添加：
+
+    ```
+    spark.tispark.use.tiflash true
+    ```
+
+2. 在启动 spark shell 或 thrift server 时，启动命令中添加 `--conf spark.tispark.use.tiflash=true`
```
Comment (ran-huang) [inline reference/tiflash/use-tiflash.md:154]:
```suggestion
2. 在启动 Spark shell 或 Thrift server 时，启动命令中添加 `--conf spark.tispark.use.tiflash=true`
```

File: reference/tiflash/use-tiflash.md [hunk 22]
```diff
@@ -0,0 +1,168 @@
...
+
+## TiFlash 支持的计算下推
+
+TiFlash 主要支持谓词、聚合下推计算，下推的计算可以帮助 TiDB 进行分布式加速。暂不支持的计算类型主要是表连接和 DISTINCT COUNT，会在后续版本逐步优化。
+
+目前 TiFlash 支持了有限的常用表达式下推，想了解具体下推的表达式，请参考[该文件](https://github.com/pingcap/tidb/blob/release-3.1/expression/expression.go#L409)。
+
+例如在聚合函数或者 WHERE 条件中包含了不在上述列表中的表达式，聚合或者相关的谓词过滤会无法进行下推。
+
+如查询遇到不支持的下推计算，则需要依赖 TiDB 完成剩余计算，可能会很大程度影响 TiFlash 加速效果。对于暂不支持表达式，将会在后续陆续加入支持，也可以联系官方沟通。
```
Comment (ran-huang) [inline reference/tiflash/use-tiflash.md:168]:
```suggestion
如查询遇到不支持的下推计算，则需要依赖 TiDB 完成剩余计算，可能会很大程度影响 TiFlash 加速效果。对于暂不支持的表达式，将会在后续陆续加入支持，也可以联系官方沟通。
```

File: reference/tiflash/use-tiflash.md (hunk 15 above)
Comment (yikeke) [reply reference/tiflash/use-tiflash.md:93]: 这个 engine 应该就是按小写写的，不用改

File: reference/tiflash/use-tiflash.md [hunk 23]
```diff
@@ -0,0 +1,168 @@
...
+{{< copyable "sql" >}}
+
+```sql
+ALTER TABLE table_name SET TIFLASH REPLICA count
+```
+
+该命令的参数说明如下：
+
+- count 表示副本数，0 表示删除。
+- location_labels 为一组由用户指定字符串用于标识 label，是为了 PD 调度的 topology 隔离，可以不填。
```
Comment (yikeke) [inline reference/tiflash/use-tiflash.md:28]:
```suggestion
- location_labels 为一组由用户指定的用于标识的字符串，是为了 PD 调度的 topology 隔离，可以不填。
```
PTAL

File: reference/tiflash/use-tiflash.md [hunk 24]
```diff
@@ -0,0 +1,168 @@
...
+
+对于相同表的多次 DDL 命令，仅保证最后一次能生效。例如：
```
Comment (yikeke) [inline reference/tiflash/use-tiflash.md:30]:
```suggestion
对于相同表的多次 DDL 命令，仅保证最后一次能生效。例如下面给出的操作 `tpch50` 表的两条 DDL 命令中，只有第二条删除副本的命令能生效：
```
这里需要说得更清楚些，我补充了下，是这个意思吗？

File: reference/configuration/pd-server/configuration-file.md [hunk 25]
```diff
@@ -236,6 +236,11 @@ PD 配置文件比命令行参数支持更多的选项。你可以在 [conf/conf
 + 关闭隔离级别检查的开关，当设置为 true 时，PD 不会通过调度来提升 Region 副本的隔离级别。
 + 默认：false
 
+### `store-balance-rate`
```
Comment (disksing) [inline reference/configuration/pd-server/configuration-file.md:239]: 👍

File: reference/tiflash/deploy.md [hunk 26]
```diff
@@ -0,0 +1,120 @@
...
+    如果希望自定义部署目录，需要配置 `data_dir` 参数，不需要则不加。如果希望多盘部署，则以逗号分隔各部署目录（注意每个 `data_dir` 目录的上级目录需要赋予 tidb 用户写权限），例如：
+
+    {{< copyable "" >}}
+
+    ```ini
+    [tiflash_servers]
+    192.168.1.1 data_dir=/data1/tiflash/data,/data2/tiflash/data
+    ```
+
+3. 按照 [TiDB Ansible 部署流程](/how-to/deploy/orchestrated/ansible.md)完成集群部署的[剩余步骤](/how-to/deploy/orchestrated/ansible.md#第-10-步调整-inventoryini-文件中的变量)。
```
Comment (yikeke) [inline reference/tiflash/deploy.md:84]:
```suggestion
3. 按照 TiDB Ansible 部署流程完成集群部署的[剩余步骤](/how-to/deploy/orchestrated/ansible.md#第-10-步调整-inventoryini-文件中的变量)。
```

File: reference/tiflash/maintain.md [hunk 27]
```diff
@@ -0,0 +1,149 @@
...
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    echo 'config show replication' | /path/to/pd-ctl -u http://<pd-ip>:<pd-port>
+    ```
+
+    预期结果为 `"enable-placement-rules": "true"`。
+
+2. 监控面板 TiFlash-Summary 下的 UpTime 检查操作系统中 TiFlash 进程是否正常。
```
Comment (yikeke) [inline reference/tiflash/maintain.md:82]:
```suggestion
2. 通过 TiFlash-Summary 监控面板下的 UpTime 检查操作系统中 TiFlash 进程是否正常。
```

File: reference/tiflash/maintain.md [hunk 28]
```diff
@@ -0,0 +1,149 @@
...
+    alter table <db-name>.<table-name> set tiflash replica 0;
+    ```
+
+2. 等待相关表的 TiFlash 副本被删除（按照[查看表同步进度](/reference/tiflash/use-tiflash.md#查看表同步进度)一节操作，查不到相关表的同步信息时即为副本被删除）。
+
+3. 在 [pd-ctl](/reference/tools/pd-control.md) (tidb-ansible 目录下的 `resources/bin` 包含对应的二进制文件) 中输入 store 命令，查看该 TiFlash 节点对应的 store id。
+
+4. 在 pd-ctl 中输入 `store delete <store_id>`，其中 <store_id> 为上一步查到的该 TiFlash 节点对应的 store id。
+
+5. 等待该 TiFlash 节点对应的 store 消失或者 state_name 变成 Tomestone 再关闭 TiFlash 进程。
```
Comment (yikeke) [inline reference/tiflash/maintain.md:56]: Tomestone -> Tombstone @ilovesoup

**Pull request merged**